pkgs/top-level/python-packages.nix

```diff
@@ -2524,6 +2524,23 @@ in {
     doCheck = false; # lazy packager
   };
 
+  csscompressor = buildPythonPackage rec {
```

Comment:
Please put this expression into a separate file in `./pkgs/development/python-modules`